helm_deploy/values-dev.yaml

```diff
@@ -25,6 +25,7 @@ generic-service:
     API_BASE_URL_COURT_SENTENCING: https://remand-and-sentencing-api-dev.hmpps.service.justice.gov.uk
     API_BASE_URL_PRISON_PERSON: https://prison-person-api-dev.prison.service.justice.gov.uk
     FEATURE_EVENT_COURT_EVENT_CHARGES-INSERTED: false
+    FEATURE_EVENT_OFFENDER_PHYSICAL_DETAILS-CHANGED: false
```

Comment:
Turned off everywhere so we can add in the subscription filter while waiting for the DPS API to be available